src/integrationTest/java/org/openstreetmap/atlas/geography/atlas/raw/OsmPbfToSlicedRawAtlasTest.java

```diff
@@ -94,7 +94,7 @@ public void testPbfToSlicedRawAtlas()
         Assert.assertEquals(0, slicedRawAtlas.numberOfEdges());
         Assert.assertEquals(0, slicedRawAtlas.numberOfAreas());
         Assert.assertEquals(648704, slicedRawAtlas.numberOfPoints());
-        Assert.assertEquals(57644, slicedRawAtlas.numberOfLines());
```

Comment:
Country borders can overlap each other. After the update in `slice` method, the country that comes first alphabetically is used for slicing first. That caused a change and one feature is now sliced once not twice. See image below for example scenario
![screen shot 2018-02-27 at 2 22 54 pm](https://user-images.githubusercontent.com/597707/36758672-fb4d146a-1bc9-11e8-8e23-18c8d02f8c5e.png)